### DISCUSSION TITLE
[Feature Request]: 在硬链接完毕后，入库前，能不能增加一个同步到网盘或者其他目录的开关或设置

### DISCUSSION THREAD

**Discussion by fishensl** [jxxghp/MoviePilot#3421] [讨论]
### 当前程序版本

1.7.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

目前目录监控插件、strm生成插件这些不能很好的关联。比如目录监控刮好了，strm生成插件不能实时和目录监控关联同步生成strm，导致目录监控入库后，实际strm文件并没有到执行时间，只能定时更新或者手动执行。
希望增加插件直接关联选项，这样目录监控刮削好后关联调用strm生成插件，生成完成后再进行入库通知。

**Comment by cddjr** [comment]
也许可以通过插件实现

**Comment by fishensl** [comment]
> 也许可以通过插件实现

插件有个问题就是时效，因为插件都是监控目录然后计划任务这样的，无法很精准完美的完成这么多工作流。

是希望在主程序添加这个工作流，这样下一步操作仅仅只是延迟10-20s而已，同事目录监控也就仅仅只是在下载完成的目录了。

整个工作流实际只是在原有下载完成》硬链接到Links》刮削入库 的基础上增加成 下载完成》硬链接到Links》Links同步到网盘目录》网盘目录生成本地strm》刮削入库。
